src/Examine/LuceneEngine/SearchResults.cs

```diff
@@ -139,19 +139,23 @@ private void DoSearch(Query query, IEnumerable<SortField> sortField, int maxResu
         /// <summary>
         /// Creates the search result from a <see cref="Lucene.Net.Documents.Document"/>
         /// </summary>
+        /// <param name="docId">The doc id of the lucene document.</param>
         /// <param name="doc">The doc to convert.</param>
         /// <param name="score">The score.</param>
         /// <returns>A populated search result object</returns>
 		[SecuritySafeCritical]
-        protected SearchResult CreateSearchResult(Document doc, float score)
+        protected SearchResult CreateSearchResult(int docId, Document doc, float score)
```

Comment:
You'll need to add an overload with this since this is now a breaking change, protected members can be used publicly.